packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -575,10 +574,10 @@ const Summary: React.FunctionComponent = () => {
 	// step to the user when in this launch flow.
 	// Allow changing site title when it's the default value or when it's an empty string.
 	React.useEffect( () => {
-		if ( ! isSiteTitleStepVisible && ! isValidSiteTitle( title ) ) {
+		if ( ! isSiteTitleStepVisible && isDefaultTitle ) {
```

Comment:
If the changes suggested in [this comment](https://github.com/Automattic/wp-calypso/pull/50311/files#r580459810) are implemented, we should probably add the `undefined` check:
```suggestion
		if ( typeof title !== 'undefined' && ! isSiteTitleStepVisible && isDefaultTitle ) {
```